Vehicle vehicle
Position pos
string timestamp
---
bool result